configs/nr-otel-collector-agent-linux.yaml

```diff
@@ -187,7 +198,14 @@ service:
         - batch
         - resourcedetection
         - resource
-        - cumulativetodelta
       exporters: [logging,otlp]
-
-  extensions: [health_check]
+    traces:
+      receivers: [otlp]
+      processors: [transform, resourcedetection, batch]
+      exporters: [logging, otlp]
+    logs:
+      receivers: [otlp, filelog]
+      processors: [transform, resourcedetection, batch]
+      exporters: [logging, otlp]
```

Comment:
Having the `verbosity: detailed` in the logging exporter and then keeping it here would push too much data to OTLP Logs. We need to either reduce verbosity or exclude it from Logs.

Reply:
I would vote to decrease the verbosity, what about using the default verbosity value; `normal`? https://github.com/open-telemetry/opentelemetry-collector/tree/main/exporter/loggingexporter#getting-started